confirm-deposit

```diff
@@ -1 +1,109 @@
-# Title
+# Confirm a Deposit to the Exchange
+
+Deposits are made in several stages. 
+One of the stages is to go through the [Exchange server](https://docs.flare.network/exchange/architecture/#architecture-of-an-exchange), which monitors for deposits and updates the Balances database. 
+Run the code below to have the Exchange server listen for deposit transactions and confirm the deposits.
+
+On a blockchain, multiple [validators](h[ttps://docs.flare.network/tech/glossary/](https://docs.flare.network/tech/validators/)) examine newly created transactions and come to an agreement about whether they are valid. 
+When they are valid, they are added to a new block on the chain and identified by a one-way cryptographic algorithm, called a _hash_. 
+The one-way hash prevents transactions from being reverted. 
+[Blocks](https://docs.flare.network/tech/glossary/) are added to the [blockchain](https://docs.flare.network/tech/glossary/) in chronological order and it may take a few blocks for the hash to be completed.
+To prevent problems, we confirm deposits several blocks back (five blocks in our code), when the blocks are most likely to be irreversible.
+
+Newly submitted deposits to our receiving address are labeled "pending."
+After five blocks have been created, valid deposits are labeled "confirmed."
+
+Run the javascript below to see pending and confirmed transactions and which block number they were added to.
+
+## One-Time Setup
+
+1. Install [node.js](https://nodejs.org/en/download/). 
+On the command line, run `node -v`  to check your installation.
+
+2. Clone the Flare repository: 
+
+`git clone https://github.com/flare-foundation/flare.git
+cd flare`
+
+See [the Flare repository on GitHub](https://github.com/flare-foundation/flare).
+
```

Comment:
The Architecture page says "Your own instance with the test server" Is this relevant here or should we just stick to running in the terminal?

Reply:
Not sure what you're asking. Which page says that? I don't see that sentence here: https://docs.flare.network/exchange/architecture

Reply:
Sorry, that looks like some corruption of "Use your own node URL." (Or maybe an unintended cut & paste.)

My question is about "Use your own node URL." So, I guess I can't try that without spinning up a node. No problem.

But...In IPFS/Filecoin, a node has a peer Identifier, which is not location based, sometimes called a cryptographic ID. is "URL" the correct term here?

Reply:
Ah, now I see it, sorry, it's the comment in the code.
Yeah, it means that the URL provided in the code is for testing purposes and in production you should spin up your own node and use its URL. It is not an ID, it is literally a URL, used to access the node, so that's the correct term here.
As you say, you would need to spin up your own node to test it, so don't worry.

Reply:
Oh. OK. And, of course, wss://coston-api.flare.network/ext/bc/C/ws is an URL.